test/corecel/math/RegulaFalsiRootFinder.test.cc

```diff
@@ -65,29 +65,30 @@ TEST(RegulaFalsi, trigometric)
 }
 
 /*!
- * Solve Itersect
+ * Solve exponential intersect.
  *
  * x(t) = t
- * y(t) = exp(t)
+ * y(t) = exp(t-1)
  *
- * Point (0,2)
- * Direction (0,-1)
+ * Point (1.5,0.5)
+ * Direction (-0.7071067812,0.7071067812)
  */
-TEST(RegulaFalsi, expomential_intersect)
+TEST(RegulaFalsi, exponential_intersect)
 {
-    double x = 0.0;
-    double y = 2.0;
-    double u = 0.0;
-    double v = -1.0;
+    double x = 1.5;
+    double y = 0.5;
+    double u = -0.7071067812;
+    double v = 0.7071067812;
```

Comment:
While you're at it (see #1333) change these to `real_type`, otherwise you're making the lambdas do work in double precision